bin/porta

```diff
@@ -404,7 +405,11 @@ module Porta
 
     def in_porta(action = :system, envs = {}, sub: nil, &block)
       run_in_porta(sub: sub) do
-        send(action, porta_common_envs.merge(envs), with_ruby_wrapper(block.call))
+        command_env = porta_common_envs.merge(envs)
+        if command_env["prometheus_multiproc_dir"]
+          FileUtils.mkdir_p command_env["prometheus_multiproc_dir"]
+        end
```

Comment:
Do you mind extracting to another method?

E.g.

```suggestion
        ensure_prometheus_multiproc_dir(command_env["prometheus_multiproc_dir"])
```

```ruby
def ensure_prometheus_multiproc_dir(prometheus_multiproc_dir)
  return unless prometheus_multiproc_dir
  FileUtils.mkdir_p prometheus_multiproc_dir
end

```

Reply:
@guicassolato , do you like it like this?